src/modules/navigator/mission_block.cpp

```diff
@@ -450,7 +450,13 @@ MissionBlock::issue_command(const mission_item_s &item)
 		vcmd.param4 = item.params[3];
 		vcmd.param5 = item.params[4];
 		vcmd.param6 = item.params[5];
-		vcmd.param7 = item.params[6];
+
+		if (item.nav_cmd == NAV_CMD_DO_SET_ROI && item.altitude_is_relative) {
```

Comment:
Not sure if this is the best way to deal with the relative alt problem.

Reply:
Looks sensible.

Reply:
It's fine for now. Eventually we need to properly use the frame.